app/lib/katello/agent/base_message.rb

```diff
@@ -30,6 +27,12 @@ def json
         }
       end
 
+      def options
```

Comment:
May be helpful to put a comment here. I could see myself wondering "why does this need to be a method?"
```suggestion
      def options # may be overridden by children
```